webapp/lib/view.dart

```diff
@@ -185,6 +189,104 @@ class ConversationPanelView {
       _messages.firstChild.remove();
     }
   }
+
+  void showAfterDateFilterPrompt(DateTime dateTime) {
+    _afterDateFilterView.showPrompt(dateTime);
+  }
+}
+
+class AfterDateFilterView {
+  DivElement panel;
+  TextAreaElement _textArea;
+
+  AfterDateFilterView() {
+    _textArea = new TextAreaElement()
+      ..classes.add('after-date-prompt__textarea')
+      ..contentEditable = 'true'
+      ..onInput.listen((e) => e.stopPropagation())
+      ..onKeyDown.listen((e) => e.stopPropagation())
+      ..onKeyUp.listen((e) => e.stopPropagation())
+      ..onKeyPress.listen((e) {
+        e.stopPropagation();
+        if (e.keyCode == KeyCode.ENTER) {
+          e.stopImmediatePropagation();
+          applyFilter();
+        }
+      });
+
+    panel = DivElement()
+      ..classes.add('after-date-prompt')
+      ..append(SpanElement()
+        ..classes.add('after-date-prompt__prompt-text')
+        ..text = 'Enter date:')
```

Comment:
Could we a bit of additional text here for the expected date format?

Reply:
Done.